linux/ca-certificates/debian/build.gradle

```diff
@@ -108,8 +112,15 @@ task checkCaCertificatesDebian(type: Test) {
 	testClassesDirs = sourceSets.packageTest.output.classesDirs
 	classpath = sourceSets.packageTest.runtimeClasspath
 
+	def containerRegistry = getContainerRegistry()
+
 	environment "PACKAGE", "adoptium-ca-certificates"
 
+	if (containerRegistry) {
```

Comment:
recommend making this more explicitly a check against empty string as opposed to implicit.